From 1fffa5f79d9a54e6b8a3f6abdd16014039699245 Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Wed, 5 Apr 2017 10:57:22 -0700
Subject: [PATCH 058/532] remoteproc: zynqmp_r5: line cannot over 80 chars

commit e1b64d1f43895d7b4ec80c9a035c054935f9aff5 from
https://github.com/Xilinx/linux-xlnx.git

This is to fix the checkpatch style warning on "line over 80
characters".

Signed-off-by: Wendy Liang <jliang@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/remoteproc/zynqmp_r5_remoteproc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/remoteproc/zynqmp_r5_remoteproc.c b/drivers/remoteproc/zynqmp_r5_remoteproc.c
index c95e3bb..3a1dd07 100644
--- a/drivers/remoteproc/zynqmp_r5_remoteproc.c
+++ b/drivers/remoteproc/zynqmp_r5_remoteproc.c
@@ -645,7 +645,8 @@ static int zynqmp_r5_remoteproc_probe(struct platform_device *pdev)
 	}
 
 	res = platform_get_resource_byname(pdev, IORESOURCE_MEM, "ipi");
-	local->ipi_base = devm_ioremap(&pdev->dev, res->start, resource_size(res));
+	local->ipi_base = devm_ioremap(&pdev->dev, res->start,
+				resource_size(res));
 	if (IS_ERR(local->ipi_base)) {
 		pr_err("%s: Unable to map IPI\n", __func__);
 		ret = PTR_ERR(local->ipi_base);
-- 
2.7.4

